### ISSUE TITLE
Unexpected end of file in PHP code blocks in MD files

### ISSUE THREAD

**Issue by Arcesilas** [AtomLinter/linter-php#143]
Hi,

In an MD file, when writing some PHP code, Linter-PHP expects `?>` instead of ```. Example:

```php
```php
<?php

someCodeHere();

doSomeMore();
// Added some spaces before ``` on next line to have it displayed
    ```
```

makes Linter PHP report an error. It is required to close the PHP code with `?>` to have it clear.

Since it's not an error to not close the PHP tag, it should not be an error when writing PHP without closing tag in a code block in markdown file.

Thanks anyway for that  package, it's really great!

**Comment by Arcanemagus** [comment]
This is actually a bug with the version of `language-gfm` bundled with Atom. It is fixed in the v1.6.0 betas, so you can either update to the Beta channel, or you can install `language-markdown` and disable `language-gfm`.

The issue is that older versions of `language-gfm` marked embedded source blocks like above as the _full_ source code (`source.php` vs. `source.embedded.php`), this leads to many programs that are expecting a full PHP file to try to run on embedded code. It's been a huge mess that I'm glad is finally almost over.

**Comment by Arcesilas** [comment]
Thanks for your clear answer!
